src/services/user.service.js

```diff
@@ -0,0 +1,52 @@
+// const { v4 } = require('uuid');
+let users = [];
+let nextId = 1;
+
+const getAll = () => {
+  return users;
+};
+
+const getById = (id) => {
+  return users.find((user) => user.id === id) || null;
+};
+
+const updateUser = ({ id, name }) => {
+  const user = getById(id);
+
+  Object.assign(user, { name });
+
+  return user;
+};
+
+const createUser = (name) => {
```

Comment:
The createUser function doesn't validate the input. It's better to check if the name is not empty and throw an error otherwise.